db_webhooks/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8.12-slim-bullseye
+
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install 
+
+COPY conf/requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY conf/config.yaml ./
+
+COPY bin/docker-entrypoint.sh ./
```

Comment:
Needs the `docker-entrypoint.sh` script to be declared as executable `chmod +x docker-entrypoint.sh`? Not sure. if necessary at all...

Reply:
Good point. Apparently there is a difference in file permissions when building Docker imgs from Windows / Linux. Added chmod for good measure.